srcs/graphics/hooks.c

```diff
@@ -0,0 +1,31 @@
+#include <graphics.h>
+
+void	movement(t_graphics *graphics, size_t y, size_t x)
```

Comment:
i would prefer WASD.......................... #GamerForLife

Reply:
lol for me is QWER #League